src/main/credentialServer/httpCredentialHelperServer.ts

```diff
@@ -192,11 +191,24 @@ export class HttpCredentialHelperServer {
     request: http.IncomingMessage,
     response: http.ServerResponse): Promise<void> {
     let stderr: string;
-    const helperPath = resources.executable(helperName);
 
     try {
+      const platform = os.platform();
+      let pathVar = process.env.PATH ?? '';
```

Comment:
I think you should just die if `process.env.PATH` is null. With this line (setting it to the empty string) you're going to end up with a leading delimiter at line 202 (`pathVar += path.delimiter + path.join(...)`). I don't know if that's correct on all the platforms, but I never see it that way.

Reply:
That's really just there to satisfy TypeScript; I don't expect it to ever be empty (and we'll have all kinds of other issues in that case).

I kinda like Kotlin's ability to do `process.env.PATH ?: throw <blah>` here, but we don't have that.

Reply:
ok